flybywiresim/a32nx
# fix(efb): introduce input component to EFB device and put it on simbrief page

## Summary of Changes

Introduce better input component that can be used across all flyPad pages. 
New component uses react hooks.

## Screenshots (if necessary)



Before:
![2021-02-03 17_55_22-Window](https://user-images.githubusercontent.com/13150438/106781335-3e5f2280-6649-11eb-9554-3737049ea16e.png)

After:
![2021-02-03 16_29_59-Window](https://user-images.githubusercontent.com/13150438/106781381-49b24e00-6649-11eb-97cd-602fc9ea0f24.png)
![2021-02-03 16_30_18-Window](https://user-images.githubusercontent.com/13150438/106781391-4cad3e80-6649-11eb-83ad-a19dea114882.png)


## References
No strong reference as it's our own implementation of UX. Designed component uses colors and styles that matches already designed components.






## Additional context

This is my first PR in this project 🤙


Discord username (if different from GitHub): arturkuma#6639

## Testing instructions


1. Go to `Company` page in EFD
1. Check if input works


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
